__tests__/app.test.js

```diff
@@ -42,29 +42,32 @@ describe("GET /api/topics", () => {
         .expect(200)
         .then(({ body }) => {
           expect(body).toBeInstanceOf(Array);
-          expect(body).toBeSorted({ descending: true });
+          expect(body).toBeSortedBy('created_at', { descending: true });
           expect(body.length).toBeGreaterThan(0);
-          body.forEach((article) => {
-            expect(typeof article).toBe("comment_id");
-            expect(article).toHaveProperty("description");
-            expect(article).toHaveProperty("title");
-            expect(article).toHaveProperty("article_img_url");
-            expect(article).toHaveProperty("comment_count");
-          });
+          expect(typeof body[0]).toBe("object");
+          expect(body[0]).toHaveProperty("article_id");
+          expect(body[0]).toHaveProperty("title");
+          expect(body[0]).toHaveProperty("body");
+          expect(body[0]).toHaveProperty("votes");
+          expect(body[0]).toHaveProperty("topic");
+          expect(body[0]).toHaveProperty("author");
+          expect(body[0]).toHaveProperty("created_at");
+          expect(body[0]).toHaveProperty("comment_count");
+
```

Comment:
Avoid changing code that has been approved on previous tickets - a forEach loop is needed here